internal/services/logz/logz_common.go

```diff
@@ -9,6 +11,9 @@ import (
 
 const TagRuleName = "default"
 
+const PlanId100gb14days = "100gb14days"
+const PlanDetails100gb14days = "100gb14days@TIDgmz7xq9ge3py"
```

Comment:
this constant isn't in the validation list, thus this won't work?

Reply:
Sorry, I do not quite understand what the validation list refers to. `PlanId100gb14days` is [used in `ValidateFunc`](https://github.com/ms-zhenhua/terraform-provider-azurerm/blob/47380f6f1ab83b6362124871d0df8d4a91a85a54/internal/services/logz/logz_monitor_resource.go#L109) of `plan_id`. `PlanDetails100gb14days` is used by the Go SDK only and won't be exposed to Provider users. There are conversion functions of [`getPlanDetails`](https://github.com/ms-zhenhua/terraform-provider-azurerm/blob/47380f6f1ab83b6362124871d0df8d4a91a85a54/internal/services/logz/logz_common.go#L137) and [`getPlanDetails`](https://github.com/ms-zhenhua/terraform-provider-azurerm/blob/47380f6f1ab83b6362124871d0df8d4a91a85a54/internal/services/logz/logz_common.go#L145) for `PlanId100gb14days` and `PlanDetails100gb14days`.